plugins/Live/lang/en.json

```diff
@@ -1,69 +1,69 @@
         "VisitorsInRealTime": "Visits in Real-time",
-        "VisitorsLastVisit": "This visitor's last visit was %s days ago.",
+        "VisitorsLastVisit": "Visited %s days ago.",
         "VisitsFrom": "%1$s%2$s visits%3$s from",
         "VisitSummary": "Spent a total of %1$s%2$s on the website%3$s, and viewed %4$s pages%5$s in %6$s visits%7$s.",
         "VisitSummaryWithActionDetails": "Spent a total of %1$s%2$s on the website%3$s, and performed %4$s actions%5$s (%6$s) in %7$s visits%8$s.",
-        "RowActionTooltipDefault": "Show Visits Log segmented by this row",
-        "RowActionTooltipWithDimension": "Show Visits Log segmented by this %s",
-        "RowActionTooltipTitle": "Open segmented Visits Log",
-        "SegmentedVisitorLogTitle": "Visits Log showing visits where %1$s is \"%2$s\"",
-        "OnClickPause": "%s is started. Click to pause.",
-        "OnClickStart": "%s is stopped. Click to start.",
+        "RowActionTooltipDefault": "Show the visit log for each segment of this row",
+        "RowActionTooltipWithDimension": "Show the visit log for each segment of this %s",
+        "RowActionTooltipTitle": "Open the segmented visit log",
+        "SegmentedVisitorLogTitle": "Visit Log showing visits where %1$s is \"%2$s\"",
+        "OnClickPause": "%s is running… Click to pause it.",
+        "OnClickStart": "%s is stopped. Click to start it.",
         "ClickToSeeAllContents": "Click to see each content interaction\/impression",
-        "VisitorLogSubcategoryHelp1": "The Visits Log shows you every visit your website receives in detail. You can see what actions each visitor has taken, how they got to your site, a bit about who they are, and more (while still complying with your local privacy regulations).",
-        "VisitorLogSubcategoryHelp2": "While other reports in Matomo show how your visitors behave at an aggregate level, the Visits Log provides granular detail. You can also use segments to narrow it down to specific types of visits to understand your visitors better.",
-        "VisitorLogSubcategoryHelp3": "Learn more in the Visits Log guide.",
-        "RealTimeHelp1": "The Visits in Real-time report shows the real-time flow of visits to your website. It includes a real-time counter of your visits and page views in the last 24 hours and the previous 30 minutes.",
+        "VisitorLogSubcategoryHelp1": "The visit log shows you every visit your website receives in detail. Find out which actions each visitor has performed, how they got to your site, a bit about who they are, and more (while still complying with your local privacy regulations).",
+        "VisitorLogSubcategoryHelp2": "While other reports in Matomo show how your visitors behave at an aggregate level, the visit log provides granular detail. You can also use segments to narrow it down to specific types of visits to understand your visitors better.",
+        "VisitorLogSubcategoryHelp3": "Learn more in the visit-log guide.",
```

Comment:
When there are triple nouns or reflective verbs, it concatenates two to preserve the meaning.
Otherwise it might be " great dogs visit log", where you don't know if dogs visited a log, or it is the visit log of dogs.